app/src/main/java/com/swent/suddenbump/model/meeting_location/Location.kt

```diff
@@ -0,0 +1,3 @@
+package com.swent.suddenbump.model.meeting_location
+
+data class Location(val latitude: Double = 0.0, val longitude: Double = 0.0, val name: String = "")
```

Comment:
I get what you're saying, but after all I think it may be better to stick with this name. The confusion is actually gone when you check the package of the class: it's Location inside LocationMeeting. I also feel like giving a longer, perhaps more suggestive name might make the code harder to read.